vm/src/builtins/weakproxy.rs

```diff
@@ -4,11 +4,17 @@ use crate::{
     class::PyClassImpl,
     common::hash::PyHash,
     function::{OptionalArg, PyComparisonValue, PySetterValue},
+<<<<<<< HEAD
```

Comment:
There's some conflicts here that need cleaning up :-)

Reply:
I was about to say… @jinnarajin  have you tried compiling this locally?

Reply:
No...